packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -305,16 +305,16 @@ class SliverChildListDelegate extends SliverChildDelegate {
   }
 }
 
-/// A base class for sliver that have multiple box children.
+/// A base class for sliver that have multiple box children that can be kept alive.
 ///
 /// Helps subclasses build their children lazily using a [SliverChildDelegate].
-abstract class SliverMultiBoxAdaptorWidget extends RenderObjectWidget {
+abstract class SliverMultiKeepAliveBoxAdaptorWidget extends RenderObjectWidget {
```

Comment:
it's weird to me that this class has a delegate since the corresponding render object doesn't have the opinion that it has multiple children or needs a delegate or anything.